lib/exception/sfException.class.php

```diff
@@ -101,20 +101,31 @@ public function printStackTrace()
       }
 
       if (sfConfig::get('sf_compressed')) {
-          ob_start('ob_gzhandler');
+        ob_start('ob_gzhandler');
       }
 
       header('HTTP/1.0 500 Internal Server Error');
     }
 
-    try
-    {
-      $this->outputStackTrace($exception);
-    }
-    catch (Exception $e)
-    {
+    if (version_compare(PHP_VERSION, '7.0.0') >= 0) {
```

Comment:
Could you please fix the CS here?
The `{` should be on a new line.
Same for the `else`.